sources/tests/Unit/Command/GenerateEntity.php

```diff
@@ -47,7 +47,7 @@ public function testExecute()
                 'schema'           => 'pomm_test',
                 'relation'         => 'alpha',
                 '--prefix-ns'      => 'Model',
-                '--prefix-dir'     => 'tmp',
+                '--prefix-dir'     => 'tmp'
```

Comment:
I think PSR indicates to leave the comma in array definitions, it is more convenient anyway.